core/commands/init.py

```diff
@@ -118,7 +119,10 @@ def on_enter_directory(self, path):
 
     def do_clone(self):
         self.window.status_message("Start cloning {}".format(self.git_url))
-        self.git("clone", self.git_url, self.suggested_git_root, working_dir='.')
```

Comment:
You can do to dry it up and nicer when we want to add more clone options
```python
self.git(
    "clone", 
    "--recursive" if self.recursive else None
    self.git_url, 
    self.suggested_git_root, 
    working_dir='.')
```

Reply:
sure

Reply:
Use `else None`, Everything else is good

Reply:
sure